src/ken.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Kensetsu KEN | SORA Docs
+head:
+  - - meta
+    - name: description
+      content: Learn about KENSETSU, SORA’s MakerDAO equivalent, involving XOR token burn, KEN token rewards, and Kensetsu’s role in the DeFi space on Polkadot
+  - - meta
+    - name: keywords
+      content: token burn, SORA, Maker DAO, stablecoin
+---
+
+# Kensetsu KEN
+
+::: info
+This token was originally proposed as [SORA RFP 60](https://github.com/sora-xor/rfps/issues/60) by Satoshi Shibarimoto.
+:::
+
+## TL;DR
+
+- All of the initial KEN tokens will be distributed to accounts that burn at least 1 million XOR between blocks 14,464,000 and 14,939,200
+- Burning less than 1 million XOR will yield no KEN tokens
+- 1% of all Kensetsu stablecoins minted will be used to buy back and burn KEN on Polkaswap
+- 80% of burned KEN burned daily will be reminted and be given to Demeter Farm for farming rewards on the XOR-KUSD pool, to incentivize liquidity
+
+![](/.gitbook/assets/ken-burn-hero.png)
+
+## Kensetsu Introduction
+
+Maker DAO is the largest DeFi app, with $5.92 Billion in TVL; Maker DAO powers DAI, the decentralized dollar-pegged stablecoin. SORA is a new world economic system aimed at high, sustainable growth. Despite SORA’s bold ambitions, it does not have a Maker DAO equivalent on its network. That’s why in 2023, Satoshi Shibarimoto, a professionally based anon, proposed KENSETSU as SORA’s version of Maker DAO that not only builds on the successes of the original protocol but also takes advantage of the amazing Polkadot tech.
```

Comment:
`professionally based anon` what does this mean? and is this relevant?

Reply:
Addressed